Traceback (most recent call last):
  File "/dafnyportfolio/src/solve.py", line 50, in <module>
    portfolio = Portfolio(args.dafny_file, args.procedure_name, args.num_instances, instances, chosen_option_selector,
  File "/dafnyportfolio/src/Portfolio.py", line 15, in __init__
    dynamic_args = self.select_dynamic_args(num_instances)
  File "/dafnyportfolio/src/Portfolio.py", line 27, in select_dynamic_args
    return self.option_selector.create_arguments(num_processes)
  File "/dafnyportfolio/src/diversification/CustomOptionsOptionSelector.py", line 15, in create_arguments
    raise RuntimeError("Stated number of calls and number of provided argument lists do not match")
RuntimeError: Stated number of calls and number of provided argument lists do not match
Traceback (most recent call last):
  File "/dafnyportfolio/src/solve.py", line 50, in <module>
    portfolio = Portfolio(args.dafny_file, args.procedure_name, args.num_instances, instances, chosen_option_selector,
  File "/dafnyportfolio/src/Portfolio.py", line 15, in __init__
    dynamic_args = self.select_dynamic_args(num_instances)
  File "/dafnyportfolio/src/Portfolio.py", line 27, in select_dynamic_args
    return self.option_selector.create_arguments(num_processes)
  File "/dafnyportfolio/src/diversification/CustomOptionsOptionSelector.py", line 15, in create_arguments
    raise RuntimeError("Stated number of calls and number of provided argument lists do not match")
RuntimeError: Stated number of calls and number of provided argument lists do not match
Traceback (most recent call last):
  File "/dafnyportfolio/src/solve.py", line 50, in <module>
    portfolio = Portfolio(args.dafny_file, args.procedure_name, args.num_instances, instances, chosen_option_selector,
  File "/dafnyportfolio/src/Portfolio.py", line 15, in __init__
    dynamic_args = self.select_dynamic_args(num_instances)
  File "/dafnyportfolio/src/Portfolio.py", line 27, in select_dynamic_args
    return self.option_selector.create_arguments(num_processes)
  File "/dafnyportfolio/src/diversification/CustomOptionsOptionSelector.py", line 15, in create_arguments
    raise RuntimeError("Stated number of calls and number of provided argument lists do not match")
RuntimeError: Stated number of calls and number of provided argument lists do not match

============================= JOB FEEDBACK =============================

NodeName=uc2n368
Job ID: 20696971
Cluster: uc2
User/Group: da1877/iti
State: COMPLETED (exit code 0)
Nodes: 1
Cores per node: 80
CPU Utilized: 00:00:22
CPU Efficiency: 0.54% of 01:08:00 core-walltime
Job Wall-clock time: 00:00:51
Memory Utilized: 266.08 MB
Memory Efficiency: 0.30% of 87.89 GB
